src/TLuaInterpreter.cpp

```diff
@@ -2568,42 +2517,37 @@ int TLuaInterpreter::moveCursor(lua_State* L)
 {
     int s = 1;
     int n = lua_gettop(L);
-    std::string a1;
+    QString windowName;
     if (n > 2) {
         if (!lua_isstring(L, s)) {
             lua_pushstring(L, "moveCursor: wrong argument type");
             lua_error(L);
             return 1;
-        } else {
-            a1 = lua_tostring(L, s);
-            s++;
         }
+        windowName = lua_tostring(L, s);
+        s++;
     }
-    int luaFrom;
+
     if (!lua_isnumber(L, s)) {
         lua_pushstring(L, "moveCursor: wrong argument type");
         lua_error(L);
         return 1;
-    } else {
-        luaFrom = lua_tointeger(L, s);
-        s++;
     }
+    int luaFrom = lua_tointeger(L, s);
+    s++;
 
-    int luaTo;
     if (!lua_isnumber(L, s)) {
         lua_pushstring(L, "moveCursor: wrong argument type");
         lua_error(L);
         return 1;
-    } else {
-        luaTo = lua_tointeger(L, s);
     }
+    int luaTo = lua_tointeger(L, s);
 
     Host& host = getHostFromLua(L);
 
-    if (a1 == "main" || n < 3) {
+    if (windowName.isEmpty() || windowName == "main") {
```

Comment:
Here you drop the `n < 3` clause, but I think it is alike to `windowName.isEmpty()` - maybe this even improves things